.ado/publish.yml

```diff
@@ -0,0 +1,48 @@
+# Publishing VSCode Extensions https://code.visualstudio.com/api/working-with-extensions/publishing-extension
+#   * $(PAT): https://stackoverflow.com/questions/58192624/publishing-vs-code-extension-via-azure-devops/58195815#58195815
+#   * Extension Manifest (package.json): https://code.visualstudio.com/api/references/extension-manifest
+
+variables:
+  Ver.Major: 0
+  Ver.Minor: 0
+  Ver.Patch: 0
```

Comment:
I assume on each upload the version will need to increment. How does this happen? What is the version numbering scheme?

Reply:
For now the manual update of these lines is assumed (upon release/publish), and manual run of the publishing pipeline. 
After an offline discussion I assume you will give an update.